scripts/dev_setup.py

```diff
@@ -35,6 +35,7 @@ def exec_command(command):
 # command modules have dependency on azure-cli-core so install this first
 exec_command('pip install -e src/azure-cli-nspkg')
 exec_command('pip install -e src/azure-cli-core')
+exec_command('pip install -e src/azure-cli-tests')
```

Comment:
1. To enable us to publish the test utility classes so that command module developer don't have to enlist entire of azure cli core repository to use them.
2. To separate the test codes from azure-cli-core to prevent core logic from leaking into the test code.